aiven/client/cli.py

```diff
@@ -1779,14 +1779,14 @@ def service__connection_pool_create(self) -> None:
     @arg.json
     def service__connection_pool_update(self) -> None:
         """Update a connection pool for a given PostgreSQL service"""
-        kwargs = dict(
-            project=self.get_project(),
-            service=self.args.service_name,
-            pool_name=self.args.pool_name,
-            dbname=self.args.dbname,
-            pool_size=self.args.pool_size,
-            pool_mode=self.args.pool_mode,
-        )
+        kwargs = {
+            "project": self.get_project(),
+            "service": self.args.service_name,
+            "pool_name": self.args.pool_name,
+            "dbname": self.args.dbname,
+            "pool_size": self.args.pool_size,
+            "pool_mode": self.args.pool_mode,
+        }
```

Comment:
Hmm, why this change? It looks OK and arguable more readable with `dict()`

Reply:
this change was because of mypy it was complaining about the previous approach

Reply:
Hmm, wondering what was the complain?

Reply:
This was fixed last week so I don't need to make the change here anymore